src/sentry/api/permissions.py

```diff
@@ -105,7 +106,11 @@ def determine_access(self, request: Request, organization):
                         extra=extra,
                     )
 
-                    raise SsoRequired(organization)
+                    after_login_redirect = request.META.get("HTTP_REFERER", "")
```

Comment:
the changes here and to `SsoRequired` is adding `after_login_redirect` which is the external redirect to the details of the exception right?

Reply:
> which is the external redirect to the details of the exception right?

not sure what you mean by that. `HTTP_REFERER` will be the page on which an API request was made. passing that in the `after_login_redirect` param will have this page they were currently on be added to the loginUrl in the `next` query parameter.

Reply:
ah i see, i misunderstood what `HTTP_REFERER` was